From 5817a16dc4f389b4d76d707a20e52beaed323cbe Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Thu, 4 Nov 2021 12:31:05 +0000
Subject: [PATCH] Disable Play services' FIDO2 unconditionally

This is needed for the time being due to this code being broken with the
current Chromium and AOSP 12.
---
 .../webauthn/Fido2CredentialRequest.java          | 15 +--------------
 1 file changed, 1 insertion(+), 14 deletions(-)

diff --git a/components/webauthn/android/java/src/org/chromium/components/webauthn/Fido2CredentialRequest.java b/components/webauthn/android/java/src/org/chromium/components/webauthn/Fido2CredentialRequest.java
index b309a2de64f36..f3d4397849e89 100644
--- a/components/webauthn/android/java/src/org/chromium/components/webauthn/Fido2CredentialRequest.java
+++ b/components/webauthn/android/java/src/org/chromium/components/webauthn/Fido2CredentialRequest.java
@@ -235,20 +235,7 @@ public class Fido2CredentialRequest implements WindowAndroid.IntentCallback {
 
     /* Initialize the FIDO2 browser API client. */
     private boolean initFido2ApiClient() {
-        if (mFido2ApiClient != null) {
-            return true;
-        }
-
-        if (!ExternalAuthUtils.getInstance().canUseGooglePlayServices(
-                    new UserRecoverableErrorHandler.Silent())) {
-            return false;
-        }
-
-        mFido2ApiClient = Fido.getFido2PrivilegedApiClient(ContextUtils.getApplicationContext());
-        if (mFido2ApiClient == null) {
-            return false;
-        }
-        return true;
+        return false;
     }
 
     // Handles a PendingIntent from the GMSCore Fido library.
